udata/core/dataservices/models.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+from udata.api_fields import field, generate_fields
+from udata.core.badges.models import BadgeMixin
+from udata.core.dataset.models import Dataset
+from udata.core.metrics.models import WithMetrics
+from udata.i18n import lazy_gettext as _
+import udata.core.contact_point.api_fields as contact_api_fields
+import udata.core.dataset.api_fields as datasets_api_fields
+
+from udata.models import db
+
+# "frequency"
+# "harvest"
+# "internal"
+# "page"
```

Comment:
I need to work on the documentation / marsalling of generated fields in the new system.
Owner / Organization fields are inside the `Owned` mixin (but there are not present in the documentation because I didn't add the `field()` wrapper there.
Yes, we need to discuss about the `uri` field (which naming is best for URI of the resource inside data.gouv.fr and URI of the Dataservice). Maybe something like `base_api_uri`?

Reply:
I added `uri` as `resource_api_uri` which seems more explicit (but I can revert to just `uri` if you prefer)

Reply:
Yes indead… Maybe something as self_api_url and self_web_url? I think the having the conflict between this URL and the base API URL indicate that the term is not precise enough to quickly understand the meaning, no?

Reply:
Renamed in 9a3242863